atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/asts/progress/BucketProgressSerializer.java

```diff
@@ -24,17 +24,17 @@
 import com.palantir.logsafe.exceptions.SafeUncheckedIoException;
 import java.io.IOException;
 
-final class BucketProgressPersister {
+final class BucketProgressSerializer {
     private final ObjectReader progressReader;
     private final ObjectWriter progressWriter;
 
-    private BucketProgressPersister(ObjectReader progressReader, ObjectWriter progressWriter) {
+    private BucketProgressSerializer(ObjectReader progressReader, ObjectWriter progressWriter) {
         this.progressReader = progressReader;
         this.progressWriter = progressWriter;
     }
 
-    static BucketProgressPersister create(ObjectMapper mapper) {
-        return new BucketProgressPersister(
+    static BucketProgressSerializer create(ObjectMapper mapper) {
```

Comment:
but... it deserialises too? :D

Reply:
This was modelled after the AtlasDB Proxy's key serializer (that also deserializes), I'll change this back to `Persister`!